src/webgl/framebuffer.js

```diff
@@ -348,16 +412,17 @@ export default class Framebuffer extends Resource {
       mask |= GL_STENCIL_BUFFER_BIT;
     }
 
+    srcX1 = srcX1 === undefined ? srcFramebuffer.width : srcX1;
+    srcY1 = srcY1 === undefined ? srcFramebuffer.height : srcY1;
+    dstX1 = dstX1 === undefined ? this.width : dstX1;
+    dstY1 = dstY1 === undefined ? this.height : dstY1;
+
     gl.bindFramebuffer(GL_READ_FRAMEBUFFER, srcFramebuffer.handle);
     gl.bindFramebuffer(GL_DRAW_FRAMEBUFFER, this.handle);
-    gl.blitFramebuffer(
-      srcX0, srcY0, srcX1, srcY1,
-      dstX0, dstY0, dstX1, dstY1,
-      mask,
-      filter
-    );
+    gl.blitFramebuffer(srcX0, srcY0, srcX1, srcY1, dstX0, dstY0, dstX1, dstY1, mask, filter);
```

Comment:
Nit: assert or warn message when 
1. mask === 0
2. srcFramebuffer.handle === this.handle